src/common/procedure/src/local.rs

```diff
@@ -914,6 +958,14 @@ mod tests {
                 }
             }
 
+            async fn rollback(&mut self, _: &Context) -> Result<()> {
+                Ok(())
+            }
+
+            fn is_support_rollback(&self) -> bool {
```

Comment:
```suggestion
            fn rollback_supported(&self) -> bool {
```
Because I saw the error name `RollbackNotSupported`, I thing we should keep the same style.